app/assets/stylesheets/application.tailwind.css

```diff
@@ -56,15 +56,15 @@ from https://tailwindcss.com/docs/adding-custom-styles
   }
 
   .button_label {
-    @apply text-sm font-medium leading-4 text-gray-500 font-montserrat;
+    @apply text-sm font-medium leading-4 text-gray-550 font-montserrat;
   }
 
   .caption {
-    @apply text-sm font-semibold leading-4 italic text-gray-500;
+    @apply text-sm font-semibold leading-4 italic text-gray-550;
   }
 
   .breadcrumb {
-    @apply text-sm font-medium leading-4 text-gray-500;
+    @apply text-sm font-medium leading-4 text-gray-550;
   }
 
```

Comment:
Why did we change the text color here ? Is it approved by Maria ?

Reply:
Yes, this is the color that Maria suggested in her Figma board. The link is in the ticket: https://umlsait.atlassian.net/browse/LRA-457

Reply:
Pull request approved.